apps/src/lib/kits/maker/PlaygroundComponents.js

```diff
@@ -26,8 +26,12 @@ import Piezo from './Piezo';
  * @returns {Promise.<Object.<String, Object>>} board components
  */
 export function createCircuitPlaygroundComponents(board) {
-  return new Promise(resolve => {
-    resolve({
+  return Promise.all([
+    initializeSoundSensor(board),
+    initializeLightSensor(board),
+    initializeThermometer(board),
+  ]).then(sensors => {
```

Comment:
If you wanted, you could here do something like `.then(([soundSensor, lightSensor, thermometer]) => {`